src/Operations.jl

```diff
@@ -110,7 +110,11 @@ end
 function is_instantiated(env::EnvCache)::Bool
     # Load everything
     pkgs = load_all_deps(env)
-    # Make sure all paths exist
+    # If the top-level project is a package, ensure it is instantiated as well
+    if env.pkg !== nothing
+        push!(pkgs, Types.PackageSpec(name=env.pkg.name, uuid=env.pkg.uuid, version=env.pkg.version, path=dirname(env.project_file)))
```

Comment:
This might also have fixed https://github.com/JuliaLang/Pkg.jl/issues/2302

Reply:
Yes!

Reply:
Not quite, because with the my testcase from https://github.com/JuliaLang/Pkg.jl/issues/2302#issuecomment-763594149 still fails since, in that case, the package is already in the `pkgs` vector. SO this code here should first search for a package with the same uuid in `pkgs` and modify that, and if it does not exist, then `push!`.

Reply:
Unless I screwed something up, for me that testcase worked.